### PR TITLE
Add trace source docs

### PR DESCRIPTION
**Pull request by frankreno**
Add trace source docs

### CODE REVIEW

**Comment by frankreno** [comment]
@pmm-sumo @sumo-drosiek mind reviewing?

**Review by pmm-sumo — APPROVED** [review]

**Comment by sumo-drosiek** [comment]
We should add note that tracing is not available for customers by default

File: website/docs/r/http_source.html.markdown [hunk 1]
```diff
@@ -24,6 +24,14 @@ resource "sumologic_http_source" "http_source" {
   }
 }
 
+resource "sumologic_http_traces_source" "http_traces_source" {
```
Comment (vsinghal13) [inline website/docs/r/http_source.html.markdown:27]: This should be `sumologic_http_source` resource itself.

Comment (frankreno) [reply website/docs/r/http_source.html.markdown:27]: Good catch up dated the wrong property

Comment (frankreno) [reply website/docs/r/http_source.html.markdown]: Fixed

**Review by vsinghal13 — APPROVED** [review]

**Pull request merged**